obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -1056,6 +1056,10 @@ void OBS_API::destroyOBS_API(void)
 	if (audioRecordingEncoder != NULL)
 		obs_encoder_release(audioRecordingEncoder);
 
+	obs_encoder_t* audioAdvancedStreamingEncoder = OBS_service::getAudioAdvancedStreamingEncoder();
```

Comment:
can be replaced with _OBS_service::setAudioAdvancedStreamingEncoder( nullptr )_ 
it also will keep _audioAdvancedStreamingEncoder_  being changed only by its _setter_ function.

Reply:
I will make another PR when this is added to staging to address this, since until now we were releasing all of them this way I believe it's better to change they all together to make it consistent.
I believe a "ReleaseEncoders" method inside the `nodeobs_service` class will be better instead letting the `nodeobs_api` class decide if it should release them or not.